src/google/protobuf/unittest_proto3.proto

```diff
@@ -140,6 +140,11 @@ message TestAllTypes {
     string oneof_string = 113;
     bytes oneof_bytes = 114;
   }
+  
+  // Sample maps
```

Comment:
I looked at that - but it imports proto2 messages. This is the downside with only supporting proto3 - almost all the test messages at least *import* proto2 :(

I could create a map_unittest_proto3.proto though which was basically the equivalent though - would you prefer that?